src/GUI/RevEng.Shared/Cli/T4Helper.cs

```diff
@@ -22,6 +22,11 @@ public static string DropT4Templates(string projectPath, CodeGenerationMode code
                 t4Version = "900";
             }
 
+            if (useEntityTypeSplitting)
+            {
+                t4Version += "_Split";
```

Comment:
I am fine with this potentially failing, plan to remove EF Core 6 and 7 support very soon anyway - I will create an issue to add VS and EF Core 9 support for this feature